sdk/monitor/monitor-opentelemetry-exporter/src/export/base.ts

```diff
@@ -108,9 +133,15 @@ export abstract class AzureMonitorBaseExporter {
           }, this._batchSendRetryIntervalMs);
           this._retryTimer.unref();
         }
+        if (!this._isStatsbeatExporter && this._statsbeatMetrics) {
+          this._statsbeatMetrics.countSuccess(endTime - startTime);
+        }
         return { code: ExportResultCode.SUCCESS };
       } else if (statusCode && isRetriable(statusCode)) {
         // Failed -- persist failed data
+        if (statusCode === 429 && !this._isStatsbeatExporter && this._statsbeatMetrics) {
```

Comment:
if _isStatsbeatExporter  is true _statsbeatMetrics should always exist, leave a single check in multiple places you have here, order is also important to avoid unnecessary checks, we need to optimize number of checks in this method as is common and will called a lot in high load apps, you also don't want to count if Statsbeat is disabled

Reply:
Removed redundant checks for `this._statsbeatMetrics` existing since if we're not exporting statsbeat envelopes (in which case `this_isStatsbeatExporter` will be false) and statsbeat is not disabled we know `this._statsbeatMetrics` should be defined.